### PR TITLE
fix(argo-workflows): Fix typo reported in #876

### PR DESCRIPTION
**Pull request by reinvantveer**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [ ] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo-workflows/templates/controller/workflow-rb.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.workflow.rbac.create -}}
-  {{- range $namespace := or .Values.singeNamespace false | ternary (list "") (append .Values.controller.workflowNamespaces (coalesce .Values.workflow.namespace .Release.Namespace) | uniq)  }}
+  {{- range $namespace := or .Values.singleNamespace false | ternary (list "") (append .Values.controller.workflowNamespaces (coalesce .Values.workflow.namespace .Release.Namespace) | uniq)  }}
```
Comment (vladlosev) [inline charts/argo-workflows/templates/controller/workflow-rb.yaml:2]: A search shows this needs to be fixed in two more places:
https://github.com/argoproj/argo-helm/blob/37277315ce669a12e9a5a66ce03ab2a6e08eb184/charts/argo-workflows/templates/controller/workflow-sa.yaml#L2
https://github.com/argoproj/argo-helm/blob/37277315ce669a12e9a5a66ce03ab2a6e08eb184/charts/argo-workflows/templates/controller/workflow-role.yaml#L2

Comment (reinvantveer) [reply charts/argo-workflows/templates/controller/workflow-rb.yaml:2]: :laughing: classic copy-paste error

Comment (reinvantveer) [reply charts/argo-workflows/templates/controller/workflow-rb.yaml:2]: Resolved in 5830d82263ace450a909d461f93e1ba806645601

**Comment by reinvantveer** [comment]
Thanks for your diligence @vladlosev

**Review by vladlosev — APPROVED** [review]
LGTM

**Pull request merged**